src/historywork/GetRemoteFileWork.cpp

```diff
@@ -52,6 +66,17 @@ void
 GetRemoteFileWork::onSuccess()
 {
     releaseAssert(mCurrentArchive);
+
+    // Get size of local file and calculate throughput as size_bytes / (now() -
+    // mDownloadStart)
+    auto size = fs::size(mLocal);
+    auto duration = std::chrono::steady_clock::now().time_since_epoch() -
+                    mDownloadStart.time_since_epoch();
+    auto durationSeconds =
+        std::chrono::duration_cast<std::chrono::seconds>(duration);
+    auto throughput = size / durationSeconds.count();
+    mBytesPerSecondRemoteFile.Mark(throughput);
```

Comment:
meter already computes rate per second, do I don't think you need to compute rate here manually. So this should be mBytesPerSecondRemoteFile.Mark(size)